solidity/contracts/liquidity-protection/LiquidityProtection.sol

```diff
@@ -369,6 +378,77 @@ contract LiquidityProtection is TokenHandler, ContractRegistryClient, Reentrancy
         else store.removePoolFromWhitelist(_poolAnchor);
     }
 
+    /**
+     * @dev adds a high tier pool
+     * can only be called by the contract owner
```

Comment:
IMO `can be called only by the contract owner`.

Reply:
copied from any other such comment we have

Reply:
Quoting you: "the fact that we have some 'bad' stuff in previous code doesn't mean that we should port it to new code".

Reply:
and it's true, but this is a comment